README.md

```diff
@@ -1 +1,46 @@
 # terraform-aws-mcaf-s3
+
+<!--- BEGIN_TF_DOCS --->
```

Comment:
Is these begin en end markers meant to be here?

Reply:
This is needed so the terraform-docs step knows where to input the docs